conf/test_route_control.py

```diff
@@ -215,9 +239,14 @@ def test_given_new_route_when_add_new_route_entry_and_mac_not_known_then_destina
     def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_then_route_is_added_in_bess(
         self,
```

Comment:
```suggestion
    @patch("conf.route_control.send_ping")
    def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_then_route_is_added_in_bess(
        self, _
```

This should fix the permission denied issue.